roles/kustomize_deploy/tasks/install_operators.yml

```diff
@@ -14,6 +14,35 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Generate values.yaml for OLM resources
+  vars:
+    cifmw_architecture_scenario: 'common/olm'
+    cifmw_ci_gen_kustomize_values_src_file: >-
+      {{
+        (
+          cifmw_kustomize_deploy_olm_source_files,
+          'values.yaml') | path_join
+      }}
+  ansible.builtin.include_role:
```

Comment:
We may like to expose a variable to allow the user to pass an entire override (`cifmw_ci_gen_kustomize_values_userdata`)

Reply:
+1

Reply:
ack!